rosidl_typesupport_connext_cpp/resource/srv__type_support.hpp.em

```diff
@@ -83,6 +83,11 @@ send_response__@(spec.srv_name)(
   void * untyped_replier,
   const rmw_request_id_t * request_header,
   const void * untyped_ros_response);
+
+ROSIDL_TYPESUPPORT_CONNEXT_CPP_PUBLIC
+const char *
+get_request_topic_name__@(spec.srv_name)(void * untyped_requester);
```

Comment:
Yeah this is expected, because in OpenSplice the "requester" object is available and I can just call `get_topic_name` (or something like that) on it. In Connext, the requester object is type erased, so I have to ask the generated code to cast it to the right type and then call that function on it.